src/Cli/dotnet/Commands/Tool/ToolCommandParser.cs

```diff
@@ -41,4 +41,5 @@ private static CliCommand ConstructCommand()
 
         return command;
     }
+
```

Comment:
nit: stray endline

Reply:
Unresolving because stray endline doesn't seem to be removed

Reply:
Oh guess it got changed while solving merge conflicts or somehting.